packages/vuetify/src/components/VAlert/VAlert.tsx

```diff
@@ -0,0 +1,203 @@
+// Styles
+import './VAlert.sass'
+
+// Components
+import { VAvatar } from '@/components/VAvatar'
+import { VBtn } from '@/components/VBtn'
+
+// Composables
+import { makeDensityProps, useDensity } from '@/composables/density'
+import { makeElevationProps, useElevation } from '@/composables/elevation'
+import { makePositionProps, usePosition } from '@/composables/position'
+import { makeRoundedProps, useRounded } from '@/composables/rounded'
+import { makeTagProps } from '@/composables/tag'
+import { makeThemeProps, useTheme } from '@/composables/theme'
+import { makeVariantProps, useVariant } from '@/composables/variant'
+import { useBorder } from '@/composables/border'
+import { useProxiedModel } from '@/composables/proxiedModel'
+import { useTextColor } from '@/composables/color'
```

Comment:
do we sort them alphabetically?

Reply:
that is alphabetical

Reply:
right, i looked at the paths, not the imported names